pkg/errorReporter/errorReporter.go

```diff
@@ -51,7 +51,7 @@ func (reporter *ErrorReporter) ReportError(error interface{}, uri string) {
 	}, uri)
 
 	if err != nil {
-		loggerUtil.Log(fmt.Sprintf("ReportError status code: %d, err: %s", statusCode, err.Error()))
+		logger.LogUtil(fmt.Sprintf("ReportError status code: %d, err: %s", statusCode, err.Error()))
```

Comment:
the only reason to use this `LogUtil` is because injecting the request logger will be a pain, both here and in other places